packages/sync/src/Actions.php

```diff
@@ -348,6 +348,12 @@ public static function do_initial_sync() {
 			return false;
 		}
 
+		// Don't start new sync if a full sync is in process.
+		$full_sync_module = \Jetpack_Sync_Modules::get_module( 'full-sync' );
```

Comment:
`Jetpack_Sync_Modules` is now deprecated; you'd need to rely on `Automattic\Jetpack\Sync\Modules` instead, by calling `use Automattic\Jetpack\Sync\Modules;` at the top of the file and using `Modules::get_module( 'full-sync' );` here.

Reply:
What @jeherve said 👍